src/i18n/en-US/index.ts

```diff
@@ -0,0 +1,11 @@
+import general from './general';
+import systemIntake from './systemIntake';
+import businessCase from './businessCase';
+
+const enUS = {
```

Comment:
I think types or data will line up with what we'll need for file organization, but agree that it's still up for discussion.